app/javascript/src/controllers/country_selection_controller.js

```diff
@@ -0,0 +1,22 @@
+import { Controller } from "stimulus";
+
+export default class extends Controller {
+  static targets = ["assessmentTypes", "selectedCountry", "selectables"];
+
+  select(e) {
+    const countryName = this.selectedCountryTarget.value;
+    const assessmentTypes = this.selectables[countryName];
+    while (this.assessmentTypesTarget.firstChild) {
```

Comment:
we can use `this.assessmentTypesTarget.childNodes.forEach( n => this.assessmentTypesTarget.removeChild(n))` instead of the while loop here.